tardis/model/tests/data/branch85_old_config.yml

```diff
@@ -1,4 +1,4 @@
-# Example YAML configuration for TARDIS
+#Config model file with branch85 density and uniform density
```

Comment:
Do you mean uniform abundance?  How can we have both branch85 and uniform density?